### ISSUE TITLE
upload many files with one command

### ISSUE THREAD

**Issue by cloudoftg** [UsergeTeam/Userge#104]
Suggestion to create a command which can upload many files from the chat to upload to gdrive with single upload. and many urls with single command. Example :  1: i paste many urls or files in the chat in then use single command to upload all at once it doesnt matter if they are queued . Please create that command if possible.

**Comment by Phyco-Ninja** [comment]
🤔 Y not using legacy way? One by one😂

**Comment by cloudoftg** [comment]
it would be easy to upload them all at once. dont you think?

**Comment by Phyco-Ninja** [comment]
No, I don't think so. If done so wouldn't it be abuse of service? Like you just flood ur uploads to servers. Hmmmmm. Doesn't sound pleasing to me. 😐 Most probably Google would ban u 😂 (dunno if these floods warns or bans exist or not🤔)

**Comment by cloudoftg** [comment]
No no it won't flood it i just mean that get rid of all work of typing !gup again again just one time and it won't matter for me if it do upload one file at a time. i mean queued system. but all it take should be one command. any thoughts now?

**Comment by rking32** [comment]
> Suggestion to create a command which can upload many files from the chat to upload to gdrive with single upload. and many urls with single command. Example : 1: i paste many urls or files in the chat in then use single command to upload all at once it doesnt matter if they are queued . Please create that command if possible.

currently you can use commands simultaneously
